pharus/dynamic_api_gen.py

```diff
@@ -24,25 +24,26 @@ def populate_api():
 """
     route_template = """
 
-@app.route('{route}', methods=['GET'])
+@app.route('{route}', methods=['GET', 'POST'])
```

Comment:
I'd suggest we only expose the REST verb if there is a valid path there with the changes proposed. Currently, this would expose `POST` endpoints for `FetchComponent` routes and `GET` for `InsertComponent` routes.

We can either make this a new variable or make a new template.